composer.json

```diff
@@ -13,7 +13,9 @@
         "sparkphp/command": "self.version"
     },
     "require": {
-        "php": ">=5.5"
+        "php": ">=5.5",
+        "equip/assist": "^0.1.2",
+        "equip/data": "^2.3"
```

Comment:
`equip/data` would depend on `equip/immutable` and this version would depend on `equip/immutable` only.